azurerm/internal/services/storage/storage_encryption_scope_resource.go

```diff
@@ -67,6 +67,12 @@ func resourceStorageEncryptionScope() *schema.Resource {
 				Optional:     true,
 				ValidateFunc: keyVaultValidate.KeyVaultChildID,
 			},
+
+			"require_infrastructure_encryption": {
```

Comment:
for consistency with the tenses used in the provider
```suggestion
			"infrastructure_encryption_required": {
```